README.md

```diff
@@ -966,6 +966,16 @@ Each `ghw.AcceleratorDevice` struct contains the following fields:
   describing the processing accelerator card. This may be `nil` if no PCI device
   information could be determined for the card.
 
+#### filters
+The `ghw.Accelerator()` function accepts a slice of filters, of type string, as parameter
+in format `[<vendor>]:[<device>][:<class>]`, (same is the _lspci_ command).
+
+Some filter examples:
+* `::0302`. Select 3D controller cards.
+* `10de::0302`. Select Nvidia (`10de`) 3D controller cards (`0302`).
+* `1da3:1060:1200`. Select Habana Labs (`1da3`) Gaudi3 (`1060`) processing accelerator cards (`1200`).
+* `1002::`. Select AMD ATI hardware.
+
```

Comment:
the filtering functionality is something we want, no questions about it.
I imagined it a bit higher level though. Like, pass a single function which takes a `pci.Device` object and return `(bool, string)` to convey if it should be accepted, and if not why. Or perhaps we can just (ab)use a `error` as return value: if `nil`, accept, if not `err.Error()` can tell us why.

Reply:
something like
```
func Accelerator(filters ...func(dev *pci.Device) bool)
```